packages/jsii-rosetta/lib/commands/extract.ts

```diff
@@ -49,16 +62,20 @@ export async function extractSnippets(
   const translatorOptions: RosettaTranslatorOptions = {
     includeCompilerDiagnostics: options.includeCompilerDiagnostics,
     assemblies: assemblies.map((a) => a.assembly),
+    tablet: await LanguageTablet.fromOptionalFile(options.outputFile),
   };
 
   const translator = options.translatorFactory
     ? options.translatorFactory(translatorOptions)
     : new RosettaTranslator(translatorOptions);
 
   if (options.cacheTabletFile) {
-    await translator.loadCache(options.cacheTabletFile);
+    await translator.addToCache(options.cacheTabletFile);
+  }
+  await translator.addToCache(options.outputFile);
+  if (translator.hasCache()) {
```

Comment:
Instead of doing this we can also always try it, and only print if the number of successful reads is > 0.

Reply:
`readFromCache` will loop through all the snippets and check to see if it exists in the cache. This seems like redundant work if there are no elements in the cache. That is why I added `hasCache`, not because of the printing.